crates/common-eventbus/src/lib.rs

```diff
@@ -49,6 +49,26 @@ impl Event {
         )
     }
 
+    pub fn bank_account_created(actor: Uuid, bank_account_id: Uuid, tenant_id: Uuid) -> Self {
+        Self::new(
+            EventData::BankAccountCreated(TenantEventDataDetails {
+                tenant_id,
+                entity_id: bank_account_id,
+            }),
+            Some(actor),
+        )
+    }
+
+    pub fn bank_account_edited(actor: Uuid, bank_account_id: Uuid, tenant_id: Uuid) -> Self {
+        Self::new(
+            EventData::BankAccountEdited(TenantEventDataDetails {
+                tenant_id,
+                entity_id: bank_account_id,
+            }),
+            Some(actor),
+        )
+    }
+
```

Comment:
as we migrated to kafka and outbox, do we need that?

Reply:
the segment handler was migrated ?

Reply:
ah I didn't do the segment part here *

Reply:
no, o ok, lets keep it 👍